### ISSUE TITLE
Special:ExportTileSheet

### ISSUE THREAD

**Issue by tomodachi94** [FTB-Gamepedia/Tilesheets#106]
It would be nice if there was a way to export tiles in a way that is compatible with Special:CreateTileSheet.

**Comment by retep998** [comment]
Maybe, but the API already makes this very easy to do with a simple program, so low priority.

**Comment by elifoster** [comment]
Do you just want to be able to download a `Tilesheet MOD.txt` file straight from the wiki? Or more than that? If it's just that, it would probably be pretty trivial to add a button to the SheetManager.

**Comment by tomodachi94** [comment]
> Do you just want to be able to download a `Tilesheet MOD.txt` file straight from the wiki? Or more than that? If it's just that, it would probably be pretty trivial to add a button to the SheetManager.

Kinda. I'm thinking of a XML-based dump format, like Special:Dump, that has all of the tiles, or tiles from mods of your choosing.

**Comment by elifoster** [comment]
What is the use case for this that is not solved by the web api?

**Comment by tomodachi94** [comment]
> What is the use case for this that is not solved by the web api?

Exporting tilesheets without using the API.

**Comment by retep998** [comment]
> > What is the use case for this that is not solved by the web api?
> 
> Exporting tilesheets without using the API.

Yes, but what is the use case for exporting tilesheets without using the API?

**Comment by tomodachi94** [comment]
> Yes, but what is the use case for exporting tilesheets without using the API?

I don't know, I just think it would be nice to have, hence the first comment about this being low priority
